REGRESSION (iOS 18): 5 WPT CSS layout tests are constantly failing (ImageOnlyFailure).

vitorroriz opened the pull request:
#### d0deb00521fc2aa767b8d92f00992f7bbdf7a0a9
<pre>
REGRESSION (iOS 18): 5 WPT CSS layout tests are constantly failing (ImageOnlyFailure).
<a href="https://bugs.webkit.org/show_bug.cgi?id=277933">https://bugs.webkit.org/show_bug.cgi?id=277933</a>
<a href="https://rdar.apple.com/133651925">rdar://133651925</a>

Reviewed by Tim Nguyen.

Allowing fuzzy tolerance to the following tests.

Tests diff by very few pixels (example: maxDifference=0-123; totalPixels=1).

I can just observe it when running our test suite. When opening the test file on
 Safari/Minibrowser I observe no apparent clipping or differences between test/expected.

I believe this is being introduced by anti-aliasing during test for some reason and it should not have user impact.

* LayoutTests/fast/text/text-underline-position-under.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-content/quotes-025.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-content/quotes-026.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-content/quotes-027.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text/text-transform/text-transform-fullwidth-004.xht:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text/text-transform/text-transform-fullwidth-005.xht:
* LayoutTests/platform/ios/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/285585@main">https://commits.webkit.org/285585@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f24795808cccffff60fc7574ae4c958a18575156

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/73138 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/52567 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/25946 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/77357 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/24376 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/60372 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/349 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/57474 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/15954 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/76205 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/47492 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/62936 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/37887 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/44131 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/20409 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/22705 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/65986 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/20764 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/79017 "Built successfully") | 
| | [⏳ 🛠 vision ](https://ews-build.webkit.org/#/builders/visionOS-2-Build-EWS "Waiting in queue, processing has not started yet") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/45 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/65908 "Passed tests") | 
| | [  ~~🛠 vision-sim~~](https://ews-build.webkit.org/#/builders/121/builds/576 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/62941 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/65190 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/16115 "Built successfully and passed tests") | [⏳ 🧪 vision-wk2 ](https://ews-build.webkit.org/#/builders/visionOS-2-Simulator-WK2-Tests-EWS "Waiting in queue, processing has not started yet") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/119/builds/7183 "Passed tests") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/417 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/446 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/431 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/448 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (approved):
Please export to WPT

vitorroriz committed:
REGRESSION (iOS 18): 5 WPT CSS layout tests are constantly failing (ImageOnlyFailure).
https://bugs.webkit.org/show_bug.cgi?id=277933
rdar://133651925

Reviewed by Tim Nguyen.

Allowing fuzzy tolerance to the following tests.

Tests diff by very few pixels (example: maxDifference=0-123; totalPixels=1).

I can just observe it when running our test suite. When opening the test file on
 Safari/Minibrowser I observe no apparent clipping or differences between test/expected.

I believe this is being introduced by anti-aliasing during test for some reason and it should not have user impact.

* LayoutTests/fast/text/text-underline-position-under.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-content/quotes-025.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-content/quotes-026.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-content/quotes-027.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text/text-transform/text-transform-fullwidth-004.xht:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text/text-transform/text-transform-fullwidth-005.xht:
* LayoutTests/platform/ios/TestExpectations:

Canonical link: https://commits.webkit.org/285585@main
